spec/i18n/placeholders_spec.rb

```diff
@@ -0,0 +1,45 @@
+describe "Placeholders in strings" do
+  it "translations preserve placeholders in strings" do
+    dirs = [Rails.root.join('locale')]
+
+    Rails::Engine.subclasses.each do |engine|
+      dir = engine.root.join('locale')
+      dirs << dir if dir.exist?
+    end
```

Comment:
@durandom @bdunne and here it is.  I would prefer if this logic were part of the Vmdb::Plugins class.  I'm envisioning a method like `Vmdb::Plugins#registered_locales_directories` or something.